appinventor/appengine/src/com/google/appinventor/client/explorer/SourceStructureExplorer.java

```diff
@@ -130,6 +130,9 @@ public void onKeyDown(KeyDownEvent event) {
         if (keyCode == KeyCodes.KEY_DELETE || keyCode == KeyCodes.KEY_BACKSPACE) {
           event.preventDefault();
           deleteItemFromTree();
+        } else if (event.isAltKeyDown() && keyCode == KeyCodes.KEY_N) {
```

Comment:
We should be careful how this maps on macOS. It may map to Cmd+N, which browsers will intercept to create a new window.

Reply:
> We should be careful how this maps on macOS. It may map to Cmd+N, which browsers will intercept to create a new window.

It looks like it maps to Opt-N on Mac. However, there's something not working with the focus of the component in the component tree. I seem to have to click the item multiple times before the shortcut works.